### DISCUSSION TITLE
flash_utils missing for ESP32

### DISCUSSION THREAD

**Discussion by hfjg** [Bodmer/TFT_eSPI#2168] [Q&A - General]
Hi everyone, I'm just experimenting with this lib on an ESP32devkit1. But all examples that use LittleFS do not compile. I get the message "flash_utils.h not found".

Can anybody please give me a hint where to find/download those flash utils or how to solve that issue?
TIA
  hfjg

**Comment by Bodmer** [comment]
TFT_eSPI does not specifically reference that header file. You need to look at the error message and identify where it is being referenced.

The only processor that I am aware of that may require that header is the ESP8266, so maybe you are trying to load another library that is not recognising the fact that your are using an ESP32.
